pkg/daemon/cluster.go

```diff
@@ -32,5 +36,81 @@ type Cluster struct {
 	ConfDirHostPath string
 }
 
+type DeploymentConfig struct {
+	Name           string
+	NodeName       string
+	Type           string
+	Labels         map[string]string
+	Volumes        []v1.Volume
+	Containers     []v1.Container
+	InitContainers []v1.Container
+	*Cluster
+}
+
+const (
```

Comment:
not only three AppName, please keep the original

Reply:
done